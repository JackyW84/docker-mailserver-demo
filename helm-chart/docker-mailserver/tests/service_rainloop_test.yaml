suite: service_rainloop
templates:
  - service_rainloop.yaml
tests:

  - it: should create rainloop service if enabled
    set:
      rainloop.enabled: true
    asserts:
      - hasDocuments:
          count: 1
        template: service_rainloop.yaml       

  - it: should not create rainloop service if disabled
    set:
      rainloop.enabled: false
    asserts:
      - hasDocuments:
          count: 0
        template: service_rainloop.yaml  