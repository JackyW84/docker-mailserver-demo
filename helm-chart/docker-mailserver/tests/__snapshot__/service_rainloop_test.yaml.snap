manifest should match snapshot:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      annotations: null
      labels:
        app: RELEASE-NAME-docker-mailserver
        chart: docker-mailserver-0.1.0
        heritage: Tiller
        release: RELEASE-NAME
      name: RELEASE-NAME-docker-mailserver-rainloop
    spec:
      ports:
      - name: http
        port: 80
        protocol: TCP
        targetPort: 8888
      selector:
        app: RELEASE-NAME-docker-mailserver-rainloop
        release: RELEASE-NAME
